core/backend/src/IModelHost.ts

```diff
@@ -85,8 +84,10 @@ export interface CrashReportingConfig {
  */
 export enum ApplicationType {
   WebAgent,
-  WebApplicationBackend,
+  WebReadonlyApp,
+  WebEditorApp,
   NativeApp,
+  MobileApp,

```

Comment:
Does 'Native' mean 'Desktop' here?

Reply:
I eliminated this enum - it wasn't necessary.